cmd/admin-handlers_test.go

```diff
@@ -68,7 +68,7 @@ func (c cmdType) toServiceSignal() serviceSignal {
 	return serviceStatus
 }
 
-func serviceSignalReceiver(cmd cmdType, t *testing.T) {
+func mockServiceSignalReceiver(cmd cmdType, t *testing.T) {
```

Comment:
My concern is the name itself, let's not use `mock`..  We can prefix it with  `test`  rather than `mock`.